### ISSUE TITLE
Incorrect error found with valid `throw` statement

### ISSUE THREAD

**Issue by u01jmg3** [squizlabs/PHP_CodeSniffer#947]
This piece of code...

```php
throw (is_object($oException)) ? $oException : new CApiManagerException(Errs::WebMailManager_AccountCreateOnLogin);
```

... fires this error

```
'throw' is a reserved keyword introduced in PHP version 5.0 and cannot be invoked as a function (T_THROW)
```

but is okay with this

```php
throw new CApiManagerException(Errs::WebMailManager_DomainDoesNotExist);
```

**Comment by robertmain** [comment]
:+1:

**Comment by gsherwood** [comment]
What coding standard are you using? I'm not aware of any included sniffs that produce an error like this.

If you are using a custom coding standard, or one provided by another developer, please contact them to look at this issue.

**Comment by u01jmg3** [comment]
It came from [`PHPCompatibility`](https://github.com/wimg/PHPCompatibility). I will log an issue with them.

**Comment by gsherwood** [comment]
> It came from PHPCompatibility. I will log an issue with them.

Thanks. I'll close this issue for now.
